src/components/list/list.scss

```diff
@@ -138,11 +138,13 @@ md-list-item {
       align-items: center;
       justify-content: flex-start;
 
-      padding: 0;
+      padding: 0 16px;
       margin: 0;
 
+      // The button should not inherit the parents background color.
+      background-color: initial;
```

Comment:
The button wrap, has the `_md-no-style` class applied, which removes all the needed stylings.

![image](https://cloud.githubusercontent.com/assets/4987015/13751475/acd4215e-ea0a-11e5-9e38-df61535f0d01.png)

And `_md-no-style` applies the background color inheritance to the button wrap, which is a bug, because the button wrap, should be transparent, as it is by default.

So now if remove the inheritance, the background-color won't be red anymore.
![image](https://cloud.githubusercontent.com/assets/4987015/13751521/e7122ab4-ea0a-11e5-8ad9-c89e904fc6fb.png)

This is a bug, because this is our `executor and ripple` wrap, which overlays over the actual list. So that means, if it has the inherited background-color, the complete list will be invisible.